rt-neural-generic/src/rt-neural-generic.cpp

```diff
@@ -381,7 +388,11 @@ void RtNeuralGeneric::connect_port(LV2_Handle instance, uint32_t port, void *dat
         case PREGAIN:
             self->pregain_db = (float*) data;
             break;
-#if AIDADSP_CONDITIONED_MODELS
+#if AIDADSP_CONDITIONED_MODELS && (AIDADSP_PARAMS == 1)
```

Comment:
there is some duplicated code here.
we can have the check be `if AIDADSP_CONDITIONED_MODELS && (AIDADSP_PARAMS >= 1)`. notice the `>=` instead of `==`. that way the same code block can be used for 1 or 2 params

Reply:
there are other small changes like this where we have duplicated code. I would say that `AIDADSP_CONDITIONED_MODELS` implies there is at least 1 conditioned parameter, so we can even skip checking for ">= 1" on that case.